values.yaml

```diff
@@ -3,7 +3,7 @@ env: dev
 services:
   expose_node_port: true
 docker:
-  registry: registry.ng.bluemix.net
+  registry: docker.io
   namespace: dlaas_dev
```

Comment:
I believe we also need to change the name space to `ffdl`?

Reply:
Lets do that in a susequent PR - once we have this working here fully

Reply:
Lets test it on Vagrant/Minikube/IBM Cloud once with this repository - and mark it as an inital release when everything is working - then we move on to adding other things

Reply:
yes

Reply:
Yeah, let's fix this in another PR. I also encountering other issues.